lee_harvey/class_scripts/temperature.py

```diff
@@ -0,0 +1,36 @@
+# Defined function to convert celsius to fahrenheit
+def convert_to_fahrenheit(v):
+    return v * 1.8 + 32
+
+# Defined function to convert fahrenheit to celsius
+def convert_to_celsius(x):
+    return (x - 32) * .5556
+
+# Provides the celsius of 100 degrees fahrenheit
+celsius_100 = convert_to_fahrenheit(100)
```

Comment:
The challenge ask you to convert from **Fahrenheit to Celsius**. the `convert_to_fahrenheit` function you defined converts **Celsius to Fahrenheit**